Gemfile.lock

```diff
@@ -104,10 +104,9 @@ GEM
       descendants_tracker (~> 0.0.1)
     colorize (0.8.1)
     concurrent-ruby (1.0.5)
-    config (1.5.0)
+    config (1.2.1)
```

Comment:
is this the result of needing a rebase on the gem refresh PR?

Reply:
Hmm Good question... rolling back seems a bad idea... I thought I had rebased, but I'll double check.